projects/fhi-angular-highcharts/src/lib/fhi-angular-highcharts.component.html

```diff
@@ -78,16 +78,16 @@ <h1 class="h3 mb-3">{{ diagramOptions.title }}</h1>
 <ng-template #diagramModal>
   <fhi-modal
     [openModalButtonClass]="'fhi-btn-icon fhi-btn-icon--circular'"
-    [modalTitle]="diagramOptions.title"
+    [modalTitle]="diagramOptionsInternal.title"
     [size]="'fullscreen'"
   >
     <ng-container fhi-modal.button>
       <i class="icon-arrows-fullscreen"></i>
       <span class="visually-hidden">Fullskjerm </span>
     </ng-container>
     <ng-container fhi-modal.body>
-      @if (diagramOptions.description) {
-        <p>{{ diagramOptions.description }}</p>
+      @if (diagramOptionsInternal.description) {
```

Comment:
@proand  `description` property is optional. Use `@if (diagramOptionsInternal?.description)`